divio_cli/localdev/main.py

```diff
@@ -387,10 +391,11 @@ def get_db_restore_command(self):
         return cmd.format(self.db_dump_path)
 
 
-def pull_media(client, stage, remote_id, path=None):
+def pull_media(client, stage, remote_id=None, path=None):
     project_home = utils.get_project_home(path)
     website_id = utils.get_aldryn_project_settings(project_home)['id']
     website_slug = utils.get_aldryn_project_settings(project_home)['slug']
+    remote_id = remote_id or website_id
     remote_project_name = (
         website_slug if remote_id == website_id
         else 'Project {}'.format(remote_id)
```

Comment:
This is interesting... I had this yesterday, and I am quite sure the remote_project_name is not adapted if the remote_id is passed in explicitly when *pulling the database*.

Reply:
Let me take a look, thanks for the notice

Reply:
Well it should work indeed: https://github.com/chescales/divio-cli/blob/82423048e350badafd4bbfcee6f157d6e641eed8/divio_cli/localdev/main.py#L349-L352